clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -120,13 +120,13 @@ We initially create a memory out of simple registers:
 
 @
 dataMem
-  :: Clock domain gated
-  -> Reset domain synchronous
-  -> Signal domain MemAddr
+  :: Clock tag
```

Comment:
What about the enable? The rest of the docs for this module seem to missing the enable as well